lib/valid_move.rb

```diff
@@ -1,4 +1,22 @@
 # code your #valid_move? method here
 
+def valid_move?(board, position)
+  if position_taken?(board, position.to_i - 1) == true
```

Comment:
`if position_taken?(board, position.to_i - 1)` is enough, ruby is implicitly truthy